create-repo

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# vim: ft=sh sw=2 et
+# shellcheck shell=bash
+set -efo pipefail
+
+# A Bash script that automates creating a Docker Hub Repository, if applicable. Also adds an org group, engineering, to have read/write access to said repo if applicable
+
+#### Requires the following parameters to be passed in from the CLI:
+# Docker Hub API Token = $1
+# User/Org name = $2
+# Repo name = $3
+
+# Checks if required values are passed in
+test -z $1 && (echo "Script needs a token towards Docker Hub!" && exit 1)
+test -z $2 && (echo "Script needs a user/org that exists in Docker Hub!" && exit 1)
+test -z $3 && (echo "Script needs a repo name to verify/use!" && exit 1)
+
+REPO_EXISTS_CODE=$(curl --write-out %{response_code} --output /dev/null -s \
+    -H "Content-Type: application/json" -H "Authorization: JWT $1" \
+    --url https://hub.docker.com/v2/repositories/$2/$3/)
+
+if [ $REPO_EXISTS_CODE -ne 200 ]; then
+    echo "Docker Hub repo $3 does not exist in $2, creating now..."
+    cat << EOF > ./data.json
+{
+    "namespace": "$2",
+    "name": "$3",
+    "description": "$(sed '2q;d' ./imgs/$3/README.md)",
+    "is_private": false
+}
+EOF
+    curl -s -H "Content-Type: application/json" -H "Authorization: JWT $1" -X POST \
+    -d @./data.json --url https://hub.docker.com/v2/repositories/
+    echo
+
+    # Clean up
+    rm ./data.json
+else
+    echo "Repo $3 already exists in $2."
```

Comment:
We should `exit 0` here if the repo already exists. I dont think we need to worry enforcing the group or readme for every publish run (Im worried about API limits as the number repos grows). Maybe we come back and add for `-f` (force flag) that does this but once we reconcile the delta with the 3 existing docker repos all the others should come into line.

Reply:
Ah, good catch as well, I don't think we'll be affected by that limit now, but nevertheless, good suggestions.  If we do do the `-f` flag or something that should be a straightforward addition, but yeah, I was thinking this on the basis of updating any existing repos with a `README.md`.